Web/Pages/Index.cshtml.cs

```diff
@@ -12,16 +12,19 @@ namespace Web.Pages
     public class IndexModel : PageModel
     {
         private readonly ILogger<IndexModel> _logger;
+        private readonly ClassificationService _classificationService;
         
+        public Classification[] Classifications;
```

Comment:
If you aren't using the classification data the references to ClassificationService should be removed.